src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -1,30 +1,137 @@
-import { Box, Text } from "@artsy/palette"
-import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
 import { FairArtworks_fair } from "v2/__generated__/FairArtworks_fair.graphql"
+import { BaseArtworkFilter } from "v2/Components/v2/ArtworkFilter"
+import { ArtworkFilterContextProvider } from "v2/Components/v2/ArtworkFilter/ArtworkFilterContext"
+import { updateUrl } from "v2/Components/v2/ArtworkFilter/Utils/urlBuilder"
+import { Match, RouterState, withRouter } from "found"
+import React from "react"
+import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
 
-interface FairArtworksProps {
+interface FairArtworksFilterProps {
   fair: FairArtworks_fair
+  relay: RelayRefetchProp
+  match?: Match
 }
 
-const FairArtworks: React.FC<FairArtworksProps> = ({ fair }) => {
+const FairArtworksFilter: React.FC<FairArtworksFilterProps> = props => {
+  const { match, relay, fair } = props
+  const { filtered_artworks } = fair
+
+  const hasFilter = filtered_artworks && filtered_artworks.id
+
+  // If there was an error fetching the filter,
+  // we still want to render the rest of the page.
+  if (!hasFilter) return null
+
   return (
-    <Box py={2}>
-      <Text variant="text">Artworks</Text>
-    </Box>
+    <ArtworkFilterContextProvider
+      filters={match && match.location.query}
+      sortOptions={[
+        { value: "-decayed_merch", text: "Default" },
+        { value: "-has_price,-prices", text: "Price (desc.)" },
+        { value: "-has_price,prices", text: "Price (asc.)" },
+        { value: "-partner_updated_at", text: "Recently updated" },
+        { value: "-published_at", text: "Recently added" },
+        { value: "-year", text: "Artwork year (desc.)" },
+        { value: "year", text: "Artwork year (asc.)" },
+      ]}
+      onChange={updateUrl}
+    >
+      <BaseArtworkFilter
```

Comment:
Now that this artwork filter is everywhere and this pattern is most prevelant I wonder if we should rename `BaseArtworkFilter` to just `ArtworkFilter` and swap out the one single use of `ArtworkFilter` with the pattern seen in this file (and elsewhere). 

Originally I was thinking we could mostly use the exported `ArtworkFilter` which wraps everything, but also wanted to export all the bits and pieces for custom configs, but in practice everything is slightly custom making me think that the exported `ArtworkFilter` is basically just indirection. If that were removed then everything would be consistent. 

(Just thinking out loud here, not suggesting for this PR!)

Reply:
I think I'm for that. Personally some of the additional configuration (the 'custom' stuff) that is happening doesn't bother me. It's pretty clear and I think forward looking (since inevitably some use cases will be custom. At the very least we've been discussing having each place a filter is used passing in its own 'default' sort, so we can have different behavior on a page where the filter is more commerce-oriented, vs. not, etc.).

That's all to say that composing a couple of things and passing those in, like we do here, feels fine.

The main lines of code that are added which feel sort of redundant/annoying are ones like https://github.com/artsy/force/blob/bc378b822c691970f14e19f87e7d73917ce7e563/src/v2/Apps/ArtistSeries/Components/ArtistSeriesArtworksFilter.tsx#L61-L101 and https://github.com/artsy/force/blob/bc378b822c691970f14e19f87e7d73917ce7e563/src/v2/Apps/ArtistSeries/Components/ArtistSeriesArtworksFilter.tsx#L111-L157 . I don't know if its worth consolidating some of the argument definitions into a constant, co-locating with the base filter, and then interpolating it where it gets used. Purely from removing some of the copy-pasta needed to set up a new instance.